lib/view/welcome/intro_slider.dart

```diff
@@ -0,0 +1,105 @@
+import 'package:flutter/material.dart';
+import 'package:intro_slider/intro_slider.dart';
+import 'package:intro_slider/slide_object.dart';
+import 'package:parkingappmobile/configs/themes/app_color.dart';
+import 'package:parkingappmobile/constants/assets_path.dart';
+
+class IntroSliderPage extends StatefulWidget {
+  const IntroSliderPage({Key? key}) : super(key: key);
+
+  @override
+  _IntroSliderPageState createState() => _IntroSliderPageState();
+}
+
+class _IntroSliderPageState extends State<IntroSliderPage> {
+  List<Slide> slides = [];
+
+  @override
+  void initState() {
+    // TODO: implement initState
+    super.initState();
+
+    slides.add(
```

Comment:
My Ideas you create array object slide and divide file  and you use forEach slides.add(item)
target: short code and easy to maintain